transcom/mymove
# B 20076 int V2

## [B-20076](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A950884)
## [B-20075](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A950883)
## [B-20074](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A950882)

## Summary

This is just to update NTSR to NTS-Release on the Move Task Order tabs estimated weight breakdown.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

B-20076
1. Access the Office app
2. Login as a TOO/TIO
3. Find the move you created
4. Navigate to the Move Task Order tab
5. Click 'Show Breakdown'
6. Verify that NTSR now displays as NTRS-Release


To test multiple iterations (some with estimated weight higher, some with actual weight higher) you can save time on creation by editing the prime_estimated_weight, prime_actual_weight, and ntsRecordedWeight values in the mto_shipment table for the shipment in question. 

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
